backend_new/src/modules/email.module.ts

```diff
@@ -0,0 +1,29 @@
+import { SendGridModule } from '@anchan828/nest-sendgrid';
```

Comment:
I think my preference is to use: https://www.npmjs.com/package/@sendgrid/mail and build our own service and module kinda like what I did https://github.com/bloom-housing/bloom/pull/3617 for twilio (sms service)

I feel like its safer with the official sendgrid package, since this package doesn't seem to be officially part of nestjs

lemme know your thoughts though!

Reply:
I'm cool with making our own. I can basically steal the code from the package we are currently using. It is of note that the package we are using is updated frequently and using the @sendgrid/mail package under the covers.

Reply:
I've pulled out the @anchan828 package. It simplified some things, so I like it!